### PR TITLE
TC 01.5.1_04 Header/Header panel validation

### PR DESCRIPTION
**Pull request by Kamal-Zh**
https://trello.com/c/mTsE5zGx/294-tc-015104-header-header-panel

### CODE REVIEW

File: tests/headerPanel.spec.js [hunk 1]
```diff
@@ -36,5 +36,28 @@ test.describe('headerPanel', () => {
...
+
+        const signInButton = page.getByRole('link', {name: 'Sign In'});
+        await expect(signInButton).toBeVisible();
+
+        const createAccount = page.getByRole('link', { name: 'Create an Account' });
+        await expect(createAccount).toBeVisible();
+        
+        const logoImage = page.locator('.logo img');
+        await expect(logoImage).toBeVisible();
+
```
Comment (S-Lanchik) [inline tests/headerPanel.spec.js:59]: I would recommend to divide this test into few separate tests. You should take only one acceptance criteria from US, and create TC according to it.

Comment (Kamal-Zh) [reply tests/headerPanel.spec.js:59]: Thanks for the feedback! Will fix that.

Comment (Kamal-Zh) [reply tests/headerPanel.spec.js:59]: @S-Lanchik could you please help resolving? Made changes and pushed but now unable to proceed.

**Review by Kamal-Zh — COMMENTED** [review]
Made a split.

**Review by alenavolakhava — APPROVED** [review]
the test is a pass, only suggestion is to remove all the comments as we were instructed on one of the lectures from Maria and there is a conflict file, so i can reapprove after it's resolved. 
Screenshot 2024-04-22 at 8 03 56 PM

**Pull request closed without merging**